invenio_oauthclient/handlers.py

```diff
@@ -234,9 +235,17 @@ def authorized_signup_handler(resp, remote, *args, **kwargs):
 
         if user is None:
             # Auto sign-up if user not found
-            user = oauth_register(account_info)
+            security_ext = current_app.extensions['security']
```

Comment:
what do you mean exactly? 
should I create a function in `utils.py` to retrieve the `confirm_register_form`?
like:
```python
def current_confirm_register_form():
    security_ext = current_app.extensions['security']
    return security_ext.confirm_register_form
```